.github/workflows/ci.yml

```diff
@@ -447,14 +459,13 @@ jobs:
                     sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
                     sudo chmod +x /usr/local/bin/docker-compose
             -   name: Install Project
-                continue-on-error: false
                 run: |
                     git clone https://github.com/spryker/docker-sdk.git ./docker
-                    docker/sdk boot deploy.ci.acceptance.dynamic-store-off.yml
+                    docker/sdk boot deploy.ci.acceptance.dynamic-store-off.robot.yml
```

Comment:
We can't use the same deploy file for robot API and UI tests. It doesn't make sense to build all the frontend for API tests.